ananas/src/main/java/iamutkarshtiwari/github/io/ananas/editimage/ImageEditorIntentBuilder.kt

```diff
@@ -141,5 +141,37 @@ class ImageEditorIntentBuilder @JvmOverloads constructor(private val context: Co
         const val FORCE_PORTRAIT = "force_portrait"
         const val EDITOR_TITLE = "editor_title"
         const val SUPPORT_ACTION_BAR_VISIBILITY = "support_action_bar_visibility"
+
+        fun getImageEditorIntent(context: Context, imageLocation: Uri, outputPath: String): Intent {
```

Comment:
better for these functions to be on our code to be more easily changeable

Reply:
Done